core/eolearn/core/eodata.py

```diff
@@ -731,3 +734,92 @@ def plot(
             config=config,
             **kwargs,
         )
+
+
+@dataclass
+class PatchGeneratorConfig:
```

Comment:
I would move this to a new module `eolearn/core/utils/testing` or `eolearn/core/utils/test`.

Reply:
moved.